### ISSUE TITLE
encrypted.google.com

### ISSUE THREAD

**Issue by CookingMonster** [palant/searchlinkfix#51]
It aint working yet on: "encrypted.google.com"
Hope it gets included also:)

**Comment by palant** [comment]
Works fine for me. Two questions for you:

1. What's your environment? In particular, which browser and which version are you using?
2. Why are you using encrypted.google.com in the first place? All of Google Search is encrypted by now, it's not clear why they still provide that subdomain.

**Comment by leedoyle** [comment]
@palant can confirm. Chromium 62, latest CleanLinks from Google Chrome Store, https://encrypted.google.com/search?q=%s set to default.

**Comment by palant** [comment]
> block JS on google.com domain

You are seeing the expected behavior then. The extension description lists this under known issues:

> This extension cannot recover links after they have been modified. This means that it doesn’t work on Google Search on smartphones (tablet version will work fine) or **if JavaScript is disabled** — Google will produce modified links from the start rather than changing them when you click the link.

Would be nice to have the OP comment but given that he didn't reply in quite a while I am going to close this report as "worksforme."

**Comment by leedoyle** [comment]
Right, but is there a workaround? After all user scripts do work with javascript blocked on google domains.

**Comment by palant** [comment]
Of course there is a work-around - an extension (or user script) could go there and rewrite the links. It's merely not a solution I am willing to maintain here, I don't have the resources to react timely to potential breakage it causes.

**Comment by EC-O-DE** [comment]
It's quite easy to redirect all Google to encrypted.google.com/*

**Comment by palant** [comment]
> It's quite easy to redirect all Google to encrypted.google.com/*

Quite possible, but why would anybody want to do this? As I said above:

> Why are you using encrypted.google.com in the first place? All of Google Search is encrypted by now, it's not clear why they still provide that subdomain.

**Comment by EC-O-DE** [comment]
Hmm, https and encryption are different.

**Comment by EC-O-DE** [comment]
I'm afraid to reveal my redirect method as my previous method was blocked by an update when devs found out. They want people to use Google like they see it best.

**Comment by palant** [comment]
> Hmm, https and encryption are different.

You are mistaken. encrypted.google.com used to be the site where Google had HTTPS enabled. Now HTTPS is used for all Google services.

**Comment by EC-O-DE** [comment]
hmm

**Comment by EC-O-DE** [comment]
> When you click on an advertiser's website and the link is HTTP rather than HTTPS, google.com will send the search terms for that specific query to the advertiser over HTTP.
> 
> encrypted.google.com Will not send the query over the less secure HTTP protocol so it has higher level of privacy.

**Comment by palant** [comment]
Yes, I have seen that text, seems to be outdated. If you happen to run a website, have a look at your logs - there haven't been any Google search queries there in a long time. Reason is, Google has the following tag in their search pages:

    <meta content="origin" name="referrer">

That's a [Referrer Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy), supported as of Chrome 56 and Firefox 50.

**Comment by EC-O-DE** [comment]
Hmm. OK... Maybe I should ditch the dot encrypted dot part then...... I guess? It would make regexp faster when needing to change only google.THIS
